.github/workflows/main.yml

```
@@ -157,48 +156,48 @@ jobs:
           source ./bin/load_environment_light.sh;
-          ./gradlew --build-cache generateSwaggerUI asciidoctor;
+          ./gradlew --build-cache generateOpenApiDocs generateSwaggerUI asciidoctor;
           ./CICD/github/upload_doc.sh
         env:
           GH_DOC_TOKEN: ${{ secrets.GH_DOC_TOKEN}}
 
-      - name : Publish Documentation (latest)
-        if : ${{ github.event.inputs.docLatest  == 'true' }}
+      - name: Publish Documentation (latest)
+        if: ${{ github.event.inputs.docLatest  == 'true' }}
         run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:1:8: Double quote to prevent globbing and word splitting

(shellcheck)